packages/site/cms-site/src/errorHandler/ErrorHandlerBoundary.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import React, { PropsWithChildren } from "react";
+
+type Props = {
+    onError?: (error: Error) => void;
```

Comment:
This can be required

Reply:
yes. Currently I removed this prop because I am trying niko's suggestion here:
https://github.com/vivid-planet/comet/pull/2281#discussion_r1699810630

If I do add it again, I will make it required.